src/message.coffee

```diff
@@ -179,6 +212,10 @@ class SlackTextMessage extends TextMessage
   # @returns {Promise<string>} - a string that can be placed into the text for this mention
   ###
   replaceUser: (client, id, mentions) ->
+    if client.robot.brain.data.users[id]?.name?
```

Comment:
again, i think you should just call a method in the client, who can return a user, but the implementation of where that user comes from (either the brain or by calling `users.info`) is abstracted away from the message